packages/console/src/assets/docs/guides/web-next-server-actions/index.ts

```diff
@@ -0,0 +1,16 @@
+import { ApplicationType } from '@logto/schemas';
+
+import { type GuideMetadata } from '../types';
+
+const metadata: Readonly<GuideMetadata> = Object.freeze({
+  name: 'Next.js (Server Actions)',
+  description:
+    "Next.js with Server Actions, take advantage of async component of React's latest features.",
```

Comment:
"take advantage of" sounds weird - how about

```suggestion
    "Next.js with Server Actions, leverages async component from the latest features of React.",
```